### PR TITLE
chore: adding unit test for unfinished validations

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes parts of #5320

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by maze-runnar** [comment]
@MeghalBisht @ranjitkshah @sachinchauhan2889

File: tests/all/unit/api/validation/test_tickets.py [hunk 1]
```diff
@@ -113,6 +113,35 @@ def test_quantity_quantity_gt_min(self):
...
+
+    def test_discount_code_pass(self):
+        """
+        Tickets Validate Price - Tests if exception is raised
+        :return:
+        """
+        schema = TicketSchema()
+        data = {}
+        original_data = {}
+        TicketSchema.validate_discount_code(schema, data, original_data)
```
Comment (MeghalBisht) [inline tests/all/unit/api/validation/test_tickets.py:153]: exception case is missing here

Comment (maze-runnar) [reply tests/all/unit/api/validation/test_tickets.py:143]: tried, but not sure if unit test can involve queries.

**Review by MeghalBisht — DISMISSED** [review]

File: tests/all/unit/api/validation/test_sessions.py [hunk 2]
```diff
@@ -22,6 +22,15 @@ def test_date_pass(self):
         }
         SessionSchema.validate_fields(schema, data, original_data)
 
+    def test_fields_pass(self):
+        """
+        Sessions Validate Date - Tests if the function runs without an exception
```
Comment (MeghalBisht) [inline tests/all/unit/api/validation/test_sessions.py:27]:
```suggestion
        Sessions Validate Message - Tests if the function runs without an exception
```

**Review by MeghalBisht — APPROVED** [review]

**Review by sachinchauhan2889 — APPROVED** [review]

**Pull request merged**